specification/_json_spec/indices.resolve_index.json

```diff
@@ -29,6 +29,10 @@
         "options": ["open", "closed", "hidden", "none", "all"],
         "default": "open",
         "description": "Whether wildcard expressions should get expanded to open or closed indices (default: open)"
+      },
+      "ignore_unavailable": {
+        "type": "boolean",
+        "description": "Ignore unavailable indexes (default: false)"
```

Comment:
The source of truth of the `_json_spec` folder is in Elasticsearch, see https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/main/resources/rest-api-spec/api/indices.resolve_index.json. Can you please open an Elasticsearch pull request? Otherwise this will change will be overwritten in our next weekly sync.